cli/helm.go

```diff
@@ -15,32 +15,32 @@ import (
 // and should not be trusted.
 var version = "0.1.0"
 
-const globalUsage = `The Kubernetes package manager
+const globalUsage = `Helm Classic - The original Kubernetes package manager
```

Comment:
I'll drop original and will go with "a" over "the".  The indefinite article more appropriately acknowledges the existence of _other_ Kubernetes package managers. I wouldn't say that's terribly important, except that the _new_ Helm is one of them!